### PR TITLE
Add glossory entry for ReplicaSet

### PR DESCRIPTION
**Pull request by ivange94**
Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6588)

### CODE REVIEW

File: _data/glossary/replica-set.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+full-link: https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/
+related:
+- replication-controller
+tags:
+- core-object
+- architecture
+short-description: >
+  ReplicaSet is the next-generation Replication Controller.
+long-description: >
+  ReplicatSet like ReplicationController ensures that a specified number of pods replicas are running at one time.
```
Comment (tonglil) [inline _data/glossary/replica-set.yml]: > ReplicatSet, like ReplicationController, ensures...

File: _data/glossary/replica-set.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  ReplicaSet supports the new set-based selector requirements as described in the labels user guide whereas a Replication Controller only supports equality-based selector requirements.
```
Comment (tonglil) [inline _data/glossary/replica-set.yml]: > ... user guide, whereas a Replication ...

File: _data/glossary/replica-set.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+full-link: https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/
+related:
+- replication-controller
+tags:
+- core-object
+- architecture
+short-description: >
+  ReplicaSet is the next-generation Replication Controller.
+long-description: >
+  ReplicatSet, like ReplicationController, ensures that a specified number of pods replicas are running at one time.
```
Comment (Bradamant3) [inline _data/glossary/replica-set.yml]: s/ReplicatSet/ReplicaSet

**Comment by ivange94** [comment]
Updated

**Comment by Bradamant3** [comment]
OK, @ivange94 -- finally merging, with thanks for your patience and contributions!

**Pull request merged**